app/src/org/commcare/models/database/user/models/EntityStorageCache.java

```diff
@@ -133,11 +133,15 @@ public static int getSortFieldIdFromCacheKey(String detailId, String cacheKey) {
         }
     }
 
+    public static void setCachedWipedPref() {
```

Comment:
I think it would be a little cleaner if this was an overloaded implementation of `setEntityCacheWipedPref()` that took no arguments (and then can also move it to be next to that method below)

Reply:
Looks like the method with arguments was only being used by the method without, so just consolidated